aidl-rust-codegen/src/binder_impl/IUBus.rs

```diff
@@ -0,0 +1,491 @@
+/*
```

Comment:
Note that this is almost completely generated Rust code via the AIDL... not really anything to review!